frame/dex/src/tests.rs

```diff
@@ -48,9 +48,17 @@ fn assets() -> Vec<u32> {
 	s
 }
 
+fn pool_assets() -> Vec<u32> {
+	// if the storage would be public:
+	// let mut s: Vec<_> = pallet_assets::pallet::PoolAsset::<Test>::iter().map(|x| x.0).collect();
+	let mut s: Vec<_> = <<Test as Config>::PoolAssets>::assets().collect();
```

Comment:
`assets()` doesn't exist on the original assets pallet, we need to add it as a separate PR.
You can see my implementation here: https://github.com/jsidorenko/substrate-dex/blob/master/frame/assets/src/impl_fungibles.rs#L271
I would additionally add a way to get the account's assets here

Reply:
We only need this for testing... so could `#[cfg(Test)]` it?

Reply:
yep, you're right!

Reply:
We don't need to resolve this in this PR.

Reply:
yep